django/core/serializers/json.py

```diff
@@ -62,7 +62,7 @@ def end_object(self, obj):
 
     def getvalue(self):
         # Grand-parent super
-        return super(PythonSerializer, self).getvalue()
+        return super().getvalue()
```

Comment:
This happened when I replaced it using `sed`. Now I have passed codebase throught that script and it looks fine now?